tests/foreman/ui/test_ldap_auth.py

```diff
@@ -140,10 +175,63 @@ def test_positive_ad_basic_roles(self):
         :expectedresults: Log in to foreman UI successfully and can access
             appropriate functional areas in UI
 
-        :caseautomation: notautomated
-
         :CaseLevel: System
         """
+        server_name = gen_string('alpha')
+        with Session(self) as session:
+            make_ldapauth(
+                session,
+                name=server_name,
+                server=self.ldap_hostname,
+                server_type=LDAP_SERVER_TYPE['UI']['ad'],
+                login_name=LDAP_ATTR['login_ad'],
+                first_name=LDAP_ATTR['firstname'],
+                surname=LDAP_ATTR['surname'],
+                mail=LDAP_ATTR['mail'],
+                account_user=self.ldap_user_name,
+                account_passwd=self.ldap_user_passwd,
+                account_basedn=self.base_dn,
+                account_grpbasedn=self.group_base_dn,
+                orgs=[entities.Organization().create().name],
+                org_select=True,
+                locations=[entities.Location().create().name],
+                loc_select=True,
+            )
+        with Session(self, user=self.ldap_user_name,
```

Comment:
maybe you can reuse or implement something similar to
https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/ui/test_adusergroup.py#L81